internal/operands/common-templates/reconcile_test.go

```diff
@@ -100,4 +102,61 @@ var _ = Describe("Common-Templates operand", func() {
 		Expect(err).ToNot(HaveOccurred())
 		ExpectResourceExists(newEditRole(), request)
 	})
+
+	It("should remove labels only from old templates", func() {
+		oldVersion := "v0.0.1"
+		err := request.Client.Create(request.Context, getTestTemplate(oldVersion, "1", namespace))
+		Expect(err).ToNot(HaveOccurred())
+		err = request.Client.Create(request.Context, getTestTemplate(oldVersion, "2", "test-namespace"))
+		Expect(err).ToNot(HaveOccurred())
+		err = request.Client.Create(request.Context, getTestTemplate(Version, "3", namespace))
```

Comment:
I think this is what makes the test fail. The testTemplate does not define any default variant which the other test checks for. Either you should put the creation into a beforeEach and clean them up in aftereach or do the cleanup in here.